docs/en/latest/discovery/nacos.md

```diff
@@ -29,8 +29,47 @@ The performance of this module needs to be improved:
 
 ### Configuration for Nacos
 
+There are two ways to configure Nacos. Single instance and Multi instance method.
+
 Add following configuration in `conf/config.yaml` :
```

Comment:
Can remove this sentence and just add "title", e.g. 

```
```yaml title="conf/config.yaml"
```

to the single and multi instances examples